requirements.txt

```diff
@@ -8,10 +8,14 @@ yamlordereddictloader==0.4.0
 beautifulsoup4==4.7.1
 humanize==0.5.1
 python-dateutil==2.7.5
-gunicorn[gevent]
 talisker==0.11.1
 responses==0.10.5
 
+# TEMP FIX
```

Comment:
This can be achieved by simply putting talisker above flask in this requirements file, I believe